app-emulation/punes/punes-0.105.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools xdg-utils
```

Comment:
One empty line below EAPI please.

Please address this: 
```
 * QA Notice: new icons were found installed but icon cache
 * has not been updated:
 *   /usr/share/icons/hicolor/32x32/apps/punes.png
 *   /usr/share/icons/hicolor/256x256/apps/punes.png
 *   /usr/share/icons/hicolor/48x48/apps/punes.png
 *   /usr/share/icons/hicolor/16x16/apps/punes.png
 * Please make sure to call xdg_icon_cache_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.
```

looks good otherwise, compiles and seems to run fine.